### ISSUE TITLE
Consider using Hoedown

### ISSUE THREAD

**Issue by devinus** [benmills/robotskirt#31]
I have revived Sundown into an improved library called Hoedown!

https://github.com/hoedown/hoedown

Here, I merged long languishing pull requests from Sundown, merged improvements from Redcarpet, merged improvements from Rinku and Houdini, greatly improved and standardized the API, and fixed many potential bugs.

Full list of changes: https://github.com/hoedown/hoedown/compare/vmg:master...master

Sundown had a rich [collection of bindings](https://github.com/vmg/sundown#bindings) and it would truly be a shame if we couldn't all benefit from a single library in the future.

**Comment by mildsunrise** [comment]
Waiting until API v2 gets released to start migrating. ;)

**Comment by timoxley** [comment]
@devinus good news https://github.com/benmills/robotskirt/issues/32#issuecomment-28625834

**Comment by mildsunrise** [comment]
I'd like to have working bindings before Hoedown v3.
The JavaScript API will change a lot. Together with the library change, it'll cause confusion.

So I suggest we deprecate this and create official Hoedown bindings for Node.
@devinus, can you create https://github.com/hoedown/node-hoedown?

**Comment by devinus** [comment]
@jmendeth Done.

**Comment by mildsunrise** [comment]
Wow, thanks! For some reason I can't push to it, though.
Maybe you should initialize it first. Dunno.

2014-03-25 18:57 GMT+01:00 Devin Torres notifications@github.com:

**Comment by mildsunrise** [comment]
I have pushed an initial commit to https://github.com/jmendeth/node-hoedown.git, can you push it to the official repo?

``` bash
git clone https://github.com/jmendeth/node-hoedown.git && cd node-hoedown
git push https://github.com/hoedown/node-hoedown.git master
```

After that, I should be able to push to it as usual.

**Comment by devinus** [comment]
Done.

**Comment by timoxley** [comment]
:+1: :+1:

**Comment by mildsunrise** [comment]
Oh, I just realized you need to add the repositories to the Contributors team. >.<

**Comment by devinus** [comment]
Done.

**Comment by mildsunrise** [comment]
Fine, I can now push to it. One last thing: could you update the repo's description to

> Node.JS bindings for Hoedown

like in the temporary repo? That'd be all.

**Comment by devinus** [comment]
OK. Also, I've added collabs to the spec repo as well.

**Comment by mildsunrise** [comment]
Yeah, spec needs some love too.

**Comment by mildsunrise** [comment]
Done! :tada: Already published in NPM, [version `0.1.0`](https://github.com/hoedown/node-hoedown) exposes all the API<sup>*</sup>
and features [complete docs](https://github.com/hoedown/node-hoedown/blob/master/doc/index.markdown) and some [examples](https://github.com/hoedown/node-hoedown/tree/master/example).

\* Except the autolinker, I'm still not familiar with its API.

PD: @devinus it'd be interesting to use the node-hoedown's docs for the official ones, especially the flag / extension descriptions.
